py-tensorboard: submission 1.6.0

emcrisostomo opened the pull request:
#### Description

<!-- Use "skip notification" (surrounded with []) to avoid notifying maintainers -->
<!-- (delete all below for minor changes) -->

###### Type(s)
<!-- update (title contains ": U(u)pdate to"), submission (new Portfile) and CVE Identifiers are auto-detected, replace [ ] with [x] to select -->

- [ ] bugfix
- [ ] enhancement
- [ ] security fix
- [x] submission

###### Tested on
<!-- Generate version information with this command in shell:
    echo "macOS $(sw_vers -productVersion) $(sw_vers -buildVersion)"; echo "Xcode $(xcodebuild -version | awk '{print $NF}' | tr '\n' ' ')"
-->
macOS 10.13.4 17E199
Xcode 9.3 9E145

###### Verification <!-- (delete not applicable items) -->
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
<!-- Please don't open a new Trac ticket if you are submitting a pull request. -->
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

pmetzger commented:
1. Does the whl file here include binaries?
2. The long description is a bit non-descriptive. It's nice if someone can tell whether they're interested in a package by reading the long description.

emcrisostomo commented:
@pmetzger:

  * Nope, this wheel file only contains python files, OS and architecture agnostic.
  * You're right about the long description: I've updated it and squashed.

If that's ok for you I'll merge it when the checks have run.

emcrisostomo commented:
@pmetzger I don't know how to deal with the errors that appear on xcode7.3:

    Error: Unable to execute port: Portfile is from the future - check date and time of your system

Is it a glitch in the build system?

pmetzger commented:
Don't worry about that error, it's probably Travis going crazy. Anyway, LGTM, feel free to merge.

emcrisostomo commented:
Thanks @pmetzger
